arch/arm/src/phy62xx/uart.c

```diff
@@ -878,10 +887,21 @@ static int pplus_uart_receive(struct uart_dev_s *dev, unsigned int *status)
 {
   uart_Ctx_t *priv = (uart_Ctx_t *)dev->priv;
   uint32_t data;
+  static uint8_t fifo_buf_store[UART_RX_BUFFER_SIZE];
+  static int buf_head;
+  static int buf_tail;
```

Comment:
why use static

Reply:
use ring_buff , need static point to the position

Reply:
But you should put the buffer into uart_Ctx_t

Reply:
BTW, please fix the mix case( e.g. uart_Ctx_t) in all source code, except the symbol come from the private static library or rom code.